dts/bindings/power/state.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2020, Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+description: Properties for power management state
+
+compatible: "pm-state"
+
+properties:
+    pm-state:
+        type: string
+        required: true
+        description: indicates a power state
+        enum:
+            - "PM_STATE_ACTIVE"
```

Comment:
will that (PM_STATE_ACTIVE) ever be used in this context? Maybe we should only list sleep states.

Reply:
The problem is keeping in sync these values with `enum pm_state`

Reply:
If we use the enum token approach that's not a problem.

I don't immediately see use of full active state in the Linux devicetree state descriptions, though I can imagine it could be useful (e.g. measures of base power usage in active mode).

Reply:
the enum token approach indeed solve this problem